graphql/src/datasources/es/index.ts

```diff
@@ -1,414 +1,44 @@
-            location: {
-              lat: orderByDistance.latitude,
-              lon: orderByDistance.longitude,
-            },
-            order: orderByDistance.order === 'DESCENDING' ? 'desc' : 'asc',
-            ignore_unmapped: true,
-          },
-        };
-      } else if (isDefined(orderByName)) {
-        query.sort = {
-          [`${searchResultField}.name.${language}.keyword`]: {
-            order: orderByName.order === 'DESCENDING' ? 'desc' : 'asc',
-            missing: '_last',
-          },
-        };
-      } else if (
-        isDefined(orderByAccessibilityProfile) &&
-        searchResultField === VENUE_SEARCH_RESULT_FIELD
-      ) {
-        query.sort = [
-          // Primary sort field (chosen accessibility profile's shortcoming count)
-          {
-            'venue.accessibility.shortcomings.count': {
-              order: 'asc',
-              nested: {
-                path: 'venue.accessibility.shortcomings',
-                filter: {
-                  term: {
-                    'venue.accessibility.shortcomings.profile':
-                      orderByAccessibilityProfile,
-                  },
-                },
-                max_children: 1,
-              },
-              missing: '_last',
-            },
-          },
-          // Secondary sort field (venue's name)
-          {
-            [`${searchResultField}.name.${language}.keyword`]: {
-              order: 'asc',
-              missing: '_last',
-            },
-          },
-        ];
-      }
-    }
-
-    // Resolve pagination
-    query = {
-      from,
-      size,
-      ...query,
-    };
-
-    return await this.post(`${es_index}/_search`, undefined, {
-      headers: { 'Content-Type': 'application/json' },
-      body: JSON.stringify(query),
-    });
+  async getQueryResults(props: getQueryResultsProps) {
+    await getQueryResults(this.post, props);
   }
 
-  async getSuggestions(
-    prefix: string,
-    languages: ElasticLanguage[],
-    size: number,
-    index: ElasticSearchIndex = ES_DEFAULT_INDEX
-  ) {
-    const query = {
-      // Hide all source fields to decrease network load
-      _source: '',
-      suggest: {
-        suggestions: {
-          prefix,
-          completion: {
-            field: 'suggest',
-            skip_duplicates: true,
-            size,
-            contexts: {
-              language: languages,
-            },
-          },
-        },
-      },
-    };
-
-    return await this.post(`${index}/_search`, undefined, {
-      headers: { 'Content-Type': 'application/json' },
-      body: JSON.stringify(query),
-    });
+  async getSuggestions(props: getSuggestionProps) {
+    await getSuggestions(this.post, props);
   }
 
-  async getAdministrativeDivisions({
-    helsinkiCommonOnly,
-  }: AdministrativeDivisionParams) {
-    const index = helsinkiCommonOnly
-      ? ES_HELSINKI_COMMON_ADMINISTRATIVE_DIVISION_INDEX
-      : ES_ADMINISTRATIVE_DIVISION_INDEX;
-    return await this.get(
```

Comment:
this.get was changed to this.post by accident?

Reply:
good catch